model/Licensing/Classes/NoAssertionLicense.md

```diff
@@ -0,0 +1,29 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# NoAssertionLicense
```

Comment:
In RDF in 2.X, this is handled as an Individual rather than a different class.  That being said, what you have here may be a better approach.

Related to the [issue I just submitted](https://github.com/spdx/spdx-3-model/issues/76).